 # Single temperature reading.

 Header header           # timestamp is the time the temperature was measured
                         # frame_id is the location of the temperature reading

 float32 NH3     # The concentration of NH3
 float32 CO     # The concentration of CO
 float32 NO2     # The concentration of NO2
 float32 C3H8     # The concentration of C3H8
 float32 C4H10     # The concentration of C4H10
 float32 CH4     # The concentration of CH4
 float32 H2     # The concentration of H2
 float32 C2H5OH     # The concentration of C2H5OH
